realm-jni/src/io_realm_internal_TableQuery.cpp

```diff
@@ -696,6 +702,12 @@ void TableQuery_StringPredicate(JNIEnv *env, jlong nativeQueryPtr, jlongArray co
         }
         else {
             Table* tbl = getTableLink(nativeQueryPtr, arr, arr_len);
```

Comment:
It is very unclear what the macro GET_ARRAY actually returns. Is there a better name for it? including these two variables?